matpower/matpower-converter/src/main/java/com/powsybl/matpower/converter/MatpowerImporter.java

```diff
@@ -340,7 +340,7 @@ public Network importData(ReadOnlyDataSource dataSource, NetworkFactory networkF
                 LOGGER.debug("MATPOWER model {}", model.getCaseName());
 
                 ContainersMapping containerMapping = ContainersMapping.create(model.getBuses(), model.getBranches(),
-                    MBus::getNumber, MBranch::getFrom, MBranch::getTo, MBranch::getR, MBranch::getX, MatpowerImporter::isTransformer,
+                    MBus::getNumber, MBranch::getFrom, MBranch::getTo, MBranch::getThirdEnd, MBranch::getR, MBranch::getX,  MatpowerImporter::isTransformer,
```

Comment:
MBranch::getThirdEnd could be removed and replaced by () -> 0

Reply:
OK, proposed modification